packages/squiggle-lang/src/ast/peggyParser.peggy

```diff
@@ -397,36 +397,36 @@ dictConstructor 'dict'
 
 // Separators
 
+statementSeparator ';'
+  = (_ ';' / _ newLineWithPossibleComment)+ _nl
```

Comment:
I hope I got this right, seems like it should be enough to capture all possible cases.

One important case this fixes is whitespace before semicolon; previously, this didn't work:
```
x = 1
  ;
y = 2
```

Reply:
Do we actively want that to work? I can't think of a time people would actively use it.